docs/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 
 ## [0.8.0] - 2018-08
 ### Added
+* Support for progressive jpeg loading. ([#877])
```

Comment:
This should be under the unreleased section of the changelog. Also, just adding the pr number is no enough. You also need to create a link. Check how it was made for others in the same file. [#888]: httpsblabla
With your pr number

Reply:
I don't see any modification to `CHANGELOG.md` in #888